### DISCUSSION TITLE
Computer remote tablet display abnormality

### DISCUSSION THREAD

**Discussion by haisong66** [rustdesk/rustdesk#7991] [General]
### Bug Description

Computer remote tablet display abnormality

### How to Reproduce

After Windows 10 remote tablet device, the original size display is very small, and the vehicle window is very blurry when opening the vehicle. Please solve this situation.

### Expected Behavior

Computer remote tablet display clear point

### Operating system(s) on local side and remote side

Windows10->1.2.4

### RustDesk Version(s) on local side and remote side

1.2.4->1.2.4

### Screenshots

https://github.com/rustdesk/rustdesk/assets/168391999/7e8ff6d9-977f-4889-aacc-35e4c3338532

![328387128-4cc45c45-0530-44b3-8cfb-24cffb130eb6](https://github.com/rustdesk/rustdesk/assets/168391999/18fc5e71-bd0c-4784-8c37-eb049e75fd2b)

**Comment by rustdesk** [comment]
Please test droidVNC-ng and upload related video.

**Reply by haisong66** [reply]
it shows good

https://github.com/rustdesk/rustdesk/assets/168391999/6c941bf1-200d-4d87-b158-077e5aef0f54

**Reply by haisong66** [reply]
The display of an Android tablet with a screen resolution lower than 1920*1080 is like this. Is this because rustdesk is not adapted?

**Reply by rustdesk** [reply]
@21pages will follow it up.

**Reply by 21pages** [reply]
@haisong66 Could you try this program to see if it can fix the issue, https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.2.4-aarch64-rm-limit.apk, and when you click `Start service`, there will be a toast which will show your device information, please give the toast information.

**Reply by haisong66** [reply]
Is there an armv7 version?

**Reply by 21pages** [reply]
https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.2.4-armv7-rm-limit.apk

**Reply by haisong66** [reply]
This version is normal

**Reply by 21pages** [reply]
https://github.com/rustdesk/rustdesk/blob/d9b96d2f3167b8ea6f4e2bc268bfed0fa7457825/flutter/android/app/src/main/kotlin/com/carriez/flutter_hbb/MainService.kt#L252

**Reply by haisong66** [reply]
The nightly version still has this problem. Haven't updated yet?

**Reply by 21pages** [reply]
It's not a bug, the resolution was divided by 2 for better speed.

**Reply by haisong66** [reply]
Can resolutions below 1920*1080 maintain the original resolution? The displayed content is indeed too small to see clearly

**Reply by 21pages** [reply]
https://github.com/rustdesk/rustdesk/pull/8002 The scale will be removed, it will be in next nightly build

**Reply by haisong66** [reply]
After deleting the ratio, I found that the speed is a bit slow. Can it be solved without changing the resolution? Or just change the ratio of the resolution below 1920*1080.

**Reply by rustdesk** [reply]
Of course, not

**Reply by haisong66** [reply]
I have tested both versions. The previous version is faster, but the version that deletes the ratio has a delay.

**Reply by haisong66** [reply]
> #8002 The scale will be removed, it will be in next nightly build


Can it be adjusted to >1920 or 1600 to reduce the display size by half? Otherwise the 1366 tablet display is too small.
